src/stores/CheckoutController.ts

```diff
@@ -1,152 +1,165 @@
-import type { CreateOrderPayload, UpdateOrderPayload } from '#types/checkout';
-import { getLocales } from '#src/services/cleeng.account.service';
-import * as checkoutService from '#src/services/checkout.service';
-import { useConfigStore } from '#src/stores/ConfigStore';
-import { useAccountStore } from '#src/stores/AccountStore';
+import type { CardPaymentData, CreateOrderPayload, Offer, Order, PaymentMethod, PaymentWithPayPalResponse, UpdateOrderPayload } from '#types/checkout';
 import { useCheckoutStore } from '#src/stores/CheckoutStore';
+import useAccount from '#src/hooks/useAccount';
+import useService from '#src/hooks/useService';
 
-export const createOrder = async (offerId: string, paymentMethodId?: number) => {
-  const { cleengSandbox, cleengId } = useConfigStore.getState().getCleengData();
+export const createOrder = async (offer: Offer, paymentMethodId?: number): Promise<unknown> => {
+  return await useAccount(async ({ customer, auth: { jwt } }) => {
+    return await useService(async ({ checkoutService, accountService, sandbox, authProviderId }) => {
+      if (!authProviderId) throw new Error('auth provider is not configured');
 
-  const { user, auth } = useAccountStore.getState();
+      const localesResponse = await accountService.getLocales(sandbox);
```

Comment:
Can Locales be hidden inside of the checkout service? Yes in Cleeng it means a service would have to call another service or there would have to be a duplicated method, but that's probably better than having to have faked out methods in INP

Reply:
updated.